apps/cnquery/cmd/plugin.go

```diff
@@ -122,14 +122,6 @@ func (c *cnqueryPlugin) RunQuery(conf *run.RunQueryConfig, runtime *providers.Ru
 		out.WriteString("[")
 	}
 
-	// FIXME: workaround for gcp-snapshot
-	// For a gcp-snapshot asset, we start with a GCP connection.
-	// This get's overridden by a filesystem connection. The filesystem connection is what we need for the scan
-	// But later, we need the GCP runtime to cleanup the snapshot disk
-	if runtime.Provider.Instance.Name == "gcp" && runtime.Provider.Connection.Name == "filesystem" {
```

Comment:
we can clean this because it happens here https://github.com/mondoohq/cnquery/blob/main/providers/gcp/provider/provider.go#L184 now